fetch/origin/no-cors.window.js

```diff
@@ -76,3 +74,99 @@ promise_test(async function () {
   assert_equals(json[0], "null");
```

Comment:
I think we concluded that this should be different as about:blank does have an origin.

Reply:
Sorry I didn't review those existing tests before. Will fix it.